uldlib/page.py

```diff
@@ -274,9 +274,13 @@ def captcha_download_links_generator(self, captcha_solve_func, print_func=print)
                     for name in ("_token_", "timestamp", "salt", "hash", "captcha_type", "_do"):
                         captcha_data[name] = parse_single(r.text, r'name="' + re.escape(name) + r'" value="([^"]*)"')
 
+                    # https://github.com/setnicka/ulozto-downloader/issues/82
+                    if captcha_image_url.startswith("//"):
+                        captcha_image_url = "https:" + captcha_image_url
```

Comment:
It would be preferable to use `urllib.parse.urljoin`. This is function specifically designed to join urls and can handle lot of strange edge cases.

Reply:
Done.